src/main/java/org/jenkinsci/test/acceptance/junit/JenkinsAcceptanceTestRule.java

```diff
@@ -51,6 +54,8 @@ public void evaluate() throws Throwable {
                 World world = World.get();
                 Injector injector = world.getInjector();
 
+                checkOnlyForInterestingPlugins(injector.getProvider(UpdateCenterMetadata.class).get());
```

Comment:
You can simply do `injector.get(UpdateCenterMetadata.class)` and Guice will be smart enough to know that it needs to use a provider.